pages/index.tsx

```diff
@@ -99,11 +101,28 @@ export default function Home(props: { users: Array<any> }) {
   };
 
   const Shifts = (props: { shifts: Array<PageObjectResponse> }) => {
+    const [showShmirotOnly, setShowShmirotOnly ] = useState<boolean>(false);
+    let moreThanOneShift=true;
```

Comment:
Please add spaces :)
```suggestion
    let moreThanOneShift = true;
```

Reply:
Change to another concept